pages/index.md

```diff
@@ -570,6 +570,7 @@
                 + [First steps with preinstalled applications](public_cloud/compute/apps_first_steps)
                 + [Starting an instance on an attached volume](public_cloud/compute/start_instance_on_attached_volume)
                 + [Shelve or pause an instance](public_cloud/compute/suspend_or_pause_an_instance)
+                + [Obtaining Source IP in Octavia](public_cloud/containers_orchestration/managed_kubernetes/Octavia_Tutorial_Add_proxy_protocol)
```

Comment:
Why is this guide in the /managed-kubernetes/ folder and now located under the Compute category in the index?
Shouldn't it be physically relocated in the `public_cloud/public_cloud_network_services` folder and under the `[Technical resources](public-cloud-network-technical-resources)` category (from line 662) in the index?

Reply:
@yomovh , any opinion on the right index location for this new guide?